### ISSUE TITLE
[CraftTweaker] Output syntax connot be null

### ISSUE THREAD

**Issue by Snownee** [codetaylor/advancedmortars#27]

**Comment by codetaylor** [comment]
You're going to have to give me more information.

**Comment by Snownee** [comment]
For example:
```
Mortar.addRecipe(["wood", "stone", "iron", "diamond"], null, 1, <minecraft:diamond>, 0.5, [<minecraft:cobblestone>]);
```
This will cause crash crash when launching

**Comment by codetaylor** [comment]
So what you're trying to do is declare a recipe that has no output except the chanced output?

**Comment by Snownee** [comment]
Yes maybe this is not a good way to declare. A map with itemstack and chance could be better.

**Comment by codetaylor** [comment]
Reproduced:
```
Caused by: java.lang.IllegalArgumentException: stack cannot be null
	at crafttweaker.mc1120.item.MCItemStack.<init>(MCItemStack.java:39) ~[CraftTweaker2-MC1120-Main-1.12-4.0.10.306.jar:?]
	at com.codetaylor.mc.advancedmortars.modules.mortar.integration.crafttweaker.mtlib.LogHelper.getStackDescription(LogHelper.java:78) ~[advanced-mortars_main/:?]
	at com.codetaylor.mc.advancedmortars.modules.mortar.integration.crafttweaker.ZenMortar$Add.getRecipeInfo(ZenMortar.java:170) ~[advanced-mortars_main/:?]
	at com.codetaylor.mc.advancedmortars.modules.mortar.integration.crafttweaker.mtlib.BaseUndoable.describe(BaseUndoable.java:30) ~[advanced-mortars_main/:?]
[...]
```

**Comment by codetaylor** [comment]
Okay, I've patched it to work around this issue. I plan to iterate on this mod sometime soon and I will most likely improve the CraftTweaker recipe integration to allow a defined number of picks from a weighted pool for recipe results (or something along those lines). It should handle `null` as a value for the main output now, but I can't guarantee that there won't be any unexpected behavior as I didn't have that use case in mind when I initially designed it.

I will look into this more soon. Thank you for bringing it to my attention.

![image](https://user-images.githubusercontent.com/1240716/35716027-fb21bba6-078a-11e8-8db3-64591fbaff6f.png)

**Comment by codetaylor** [comment]
Fixed in 1.12.2-1.6.18
